pallets/ds-accounts/src/lib.rs

```diff
@@ -66,7 +66,9 @@ impl<
     }
 
     pub fn is_role_correct(_role: AccountRole) -> bool {
-        true
+        !(_role & ADMIN_ROLE.into()).is_zero() ||
```

Comment:
Is's too complex expression.  It can be simplified by cobining all roles in one value.